MinPlatformPkg/Test/Library/TestPointCheckLib/TestPointImageDump.h

```diff
@@ -0,0 +1,36 @@
+/** @file
+
+Copyright (c) 2017, Intel Corporation. All rights reserved.<BR>
+Copyright (c) Microsoft Corporation.
+SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#include <Uefi.h>
+#include <Protocol/LoadedImage.h>
+#include <Protocol/DevicePath.h>
+
+#ifndef _TEST_POINT_IMAGE_DUMP_H_
+#define _TEST_POINT_IMAGE_DUMP_H_
```

Comment:
Please wrap the include guard around the includes as well.

Reply:
Updated.